packages/functional-tests/lib/fixtures/standard.ts

```diff
@@ -150,47 +163,56 @@ export const test = base.extend<TestOptions, WorkerOptions>({
     });
   },
 
-  emailOptions: [{ prefix: '', PASSWORD }], // Default options for the fixture
+  emailOptions: [{ prefix: SIGNIN_EMAIL_PREFIX, password: PASSWORD }], // Default options for the fixture
 
   emails: async (
     { target, pages: { login }, emailOptions },
     use
   ): Promise<void> => {
-    const emails = await Promise.all(
-      emailOptions.map(async (emailOptions) =>
-        login.createEmail(emailOptions.prefix ?? '')
-      )
-    );
+    const emails = emailOptions.map((emailOptions) => {
+      return (
+        emailOptions.prefix.replace('{id}', Math.random() + '') +
+        '@restmail.net'
+      );
+    });
     await login.clearCache(); // Clear cache for each email
 
     // Pass the generated email to the test along with the password
     await use(emails);
+
     for (const [index, email] of emails.entries()) {
-      const emailExists = await target.auth.accountStatusByEmail(email);
-      if (emailExists.exists) {
-        const x = emailOptions[index];
-        await teardownEmail(target, email, x.NEW_PASSWORD ?? x.PASSWORD);
-      }
+      await teardownEmail(target, email, emailOptions[index]);
     }
   },
 });
 
 export async function teardownEmail(
   target: BaseTarget,
   email: string,
-  PASSWORD: string
+  emailOptions: EmailFixtureOptions
 ) {
-  try {
-    const creds = await target.auth.signIn(email, PASSWORD);
-    await target.auth.accountDestroy(email, PASSWORD, {}, creds.sessionToken);
-  } catch (error) {
-    // If the account is not created during the execution of the test, which
-    // can happen if the test fails prematurly, an exception will be thrown
-    // during email cleanup
-    const ERROR_1 = 'Bad Request';
-    const ERROR_2 = 'Request blocked';
-    if (!(error.error === ERROR_1) && !(error.error === ERROR_2)) {
-      throw error;
+  if (emailOptions.prefix == BLOCKED_EMAIL_PREFIX) {
+    return;
+  }
```

Comment:
Is there a way to properly dispose of blocked email accounts?

Reply:
A workaround was applied: https://github.com/mozilla/fxa/pull/16685/commits/049b09d5a500b0cc980fa9bac20e9b63b42133fe